backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/CreateVolumeContainerCommand.java

```diff
@@ -81,7 +81,8 @@ private CreateVolumeVDSCommandParameters getCreateVDSCommandParameters() {
     }
 
     private VolumeType getType() {
-        if (getParentParameters() != null && getParentParameters().getParentCommand() == ActionType.ConvertDisk) {
+        //When incremental flag is up there is now need to change the type of the volume
```

Comment:
Some comment's improvements:
1) Add a space before `When`.
2) `incremental flag` --> `incremental backup flag`.
3) `.` at the end of the sentence.